tests/bpftrace.cpp

```diff
@@ -265,7 +267,8 @@ TEST(bpftrace, add_probes_wildcard_no_matches)
       "kprobe:sys_read,kprobe:not_here_*,kprobe:sys_write{}");
 
   auto bpftrace = get_strict_mock_bpftrace();
-  EXPECT_CALL(*bpftrace->mock_probe_matcher, get_symbols_from_traceable_funcs())
+  EXPECT_CALL(*bpftrace->mock_probe_matcher,
```

Comment:
Worth adding a kfunc test to expect a `get_symbols_from_traceable_funcs(true)`

Reply:
Not really, there's too many things mocked in testing kfuncs and the test wouldn't have much value. OTOH, #2879 should also use `get_symbols_from_traceable_funcs(true)`, so the test can be added there.

Reply:
Ah, no, I meant that they can be added as a part of that PR.

Reply:
That was merged already (by you) :)

Reply:
My GitHub comments today really suck :-D Sorry about that.

I meant to tag #2897 which I believe should call `get_symbols_from_traceable_funcs(true)` for attachment/listing of wildcarded `kprobe:module:function` probes. That should also add tests for this code path.

Reply:
Haha no worries. All cleared up 👍